frontend/src/scenes/session-recordings/errors/SessionRecordingErrors.tsx

```diff
@@ -0,0 +1,78 @@
+import { IconFeatures } from '@posthog/icons'
+import { LemonButton, LemonCollapse, Spinner } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { urls } from 'scenes/urls'
+
+import { sessionRecordingErrorsLogic } from './sessionRecordingErrorsLogic'
+
+export function SessionRecordingErrors(): JSX.Element {
+    const { errors, errorsLoading } = useValues(sessionRecordingErrorsLogic)
+    const { loadErrorClusters } = useActions(sessionRecordingErrorsLogic)
+
+    if (errorsLoading) {
+        return <Spinner />
+    }
+
+    if (!errors) {
+        return (
+            <LemonButton size="large" type="primary" icon={<IconFeatures />} onClick={() => loadErrorClusters()}>
+                Magically find errors
```

Comment:
```suggestion
                Automagically find errors
```

I like the wordplay, feel free to ignore it.

Reply:
The irony is that the big button might not even be needed in production it'll be so "magic"

Reply:
I love "automagically" it's my favourite silliness